manager.go

```diff
@@ -617,9 +613,20 @@ func (m *Manager) InitWithOptions(elf io.ReaderAt, options Options) error {
 	for _, excludedFuncName := range m.options.ExcludedFunctions {
 		delete(m.collectionSpec.Programs, excludedFuncName)
 	}
+	for i := range m.Probes {
+		if slices.Contains(m.options.ExcludedFunctions, m.Probes[i].EBPFFuncName) {
```

Comment:
Does this really work to delete from the currently iterated slice ? It feels like it shouldn't

Reply:
You are right, this doesn't work. It iterates over all the original indices. Update incoming.

Reply:
Thanks for this PR by the way !